app/plugins/basic-authentication.md

```diff
@@ -87,6 +87,25 @@ form parameter             | default | description
 `username`                 |         | The username to use in the Basic Authentication
 `password`<br>*optional*   |         | The password to use in the Basic Authentication
 
+### Using the Credential
```

Comment:
This section should be added to the sidebar by updating the YAML section at the top of this file.